core/src/main/scala/com/ovoenergy/meters4s/Reporter.scala

```diff
@@ -279,11 +299,18 @@ private class MeterRegistryReporter[F[_]](
           def record(d: FiniteDuration) =
             F.delay(t.record(d.toMillis, MILLISECONDS))
 
+          def start() = F.delay {
+            val internal = micrometer.Timer.start(mx)
+            new Sample[F] {
+              def stop(): F[Long] = F.delay(internal.stop(t))
+            }
+          }
```

Comment:
What is gonna happen if we never call stop? I believe we will have an orphan `micrometer.Timer.start(mx)` around? Would it be a problem?

Reply:
It would leave that around, though I think that is the case with raw micrometer as well? So far as I can tell should still be GCed?

Reply:
I should GCed...I believe as I don't know it internally.